<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detalle Materia Prima</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{

	
	background-color: rgb(6, 191, 249);

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_detalle">
      <attribute name="title">
       <string>Detalle</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_codigo_det">
           <property name="text">
            <string>Código</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_codigo_det">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_nombre_det">
           <property name="text">
            <string>Nombre</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_nombre_det"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_cantidad_det">
           <property name="text">
            <string>Cantidad</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_cantidad_det">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_borrar_det">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(254, 20, 20);</string>
           </property>
           <property name="text">
            <string>Borrar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_actualizar_det">
           <property name="text">
            <string>Actualizar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="Lotes">
          <attribute name="title">
           <string>Lotes</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QTableView" name="tv_lotes_det">
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Entradas">
          <attribute name="title">
           <string>Entradas</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,5,1">
               <item>
                <widget class="QLineEdit" name="le_buscar_entrada">
                 <property name="placeholderText">
                  <string>Buscar por lote</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btn_modificar_entrada">
                 <property name="text">
                  <string>Modificar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableView" name="tv_entradas_mp_det">
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Proveedores">
          <attribute name="title">
           <string>Proveedores</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTableView" name="tv_provs_detalle_mp">
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>9</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cerrar_det">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(22, 183, 40);</string>
           </property>
           <property name="text">
            <string>Salir</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_entrada">
      <attribute name="title">
       <string>Nueva Entrada</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QFrame" name="frame_exterior">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QFrame" name="frame_form">
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="lbl_fecha_entrada">
                 <property name="text">
                  <string>Fecha</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="le_f_entrada">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="lbl_codigo_entrada">
                 <property name="text">
                  <string>Código</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="le_codigo_entrada">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="lbl_nombre_entrada">
                 <property name="text">
                  <string>Nombre</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="le_nombre_entrada">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="lbl_proveedor_entrada">
                 <property name="text">
                  <string>Proveedor</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="cb_proveedor_entrada"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="lbl_fecha_caducidad">
                 <property name="text">
                  <string>Fecha caducidad</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QDateEdit" name="de_f_caducidad">
                 <property name="displayFormat">
                  <string>d/M/yyyy</string>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                 <property name="date">
                  <date>
                   <year>2025</year>
                   <month>1</month>
                   <day>1</day>
                  </date>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="lbl_cantidad_entrada">
                 <property name="text">
                  <string>Cantidad</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="le_cantidad_entrada">
                 <property name="inputMethodHints">
                  <set>Qt::InputMethodHint::ImhDigitsOnly</set>
                 </property>
                 <property name="placeholderText">
                  <string>Campo obligatorio</string>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="lbl_ubicacion_entrada">
                 <property name="text">
                  <string>Ubicación</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QComboBox" name="cb_ubicaciones_nueva_entrada"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_cerrar_entrada">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(22, 183, 40);</string>
              </property>
              <property name="text">
               <string>Salir</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_guardar_entrada">
              <property name="text">
               <string>Guardar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
